docs/getting-started.md

```diff
@@ -0,0 +1,109 @@
+# Getting started
+
+## Prerequisites
+
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+1. Install `kustomize` `v3.1.0+` (see [kustomize/releases](https://github.com/kubernetes-sigs/kustomize/releases).
+1. Download the latest `v0.2.x` release of `clusterctl` from [cluster-api/releases](https://github.com/kubernetes-sigs/cluster-api/releases)
+1. You can use either a VM, container or existing Kubernetes cluster as bootstrap cluster.
+   - If you want to use VM, install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), version 0.30.0 or greater. Also install a [driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md). For Linux, we recommend `kvm2`. For MacOS, we recommend `VirtualBox`.
+   - If you want to use a container, install [Kind](https://github.com/kubernetes-sigs/kind#installation-and-usage).
+   - If you want to use an existing Kubernetes cluster, prepare a kubeconfig which for this cluster.
+
+## Cluster Creation
+
+1. Generate example YAML files if needed. You can use the `examples/generate.sh` script as documented [here](./../examples/README.md).
+
+2. Create a cluster:
+   - If you are using Minikube:
+
+   ```bash
+   clusterctl create cluster \
+        --bootstrap-type minikube --bootstrap-flags kubernetes-version=v1.15.0 \
+        -c examples/_out/cluster.yaml \
+        -m examples/_out/machines.yaml \
+        -p examples/_out/provider-components.yaml \
+        -a examples/addons.yaml
+   ```
+
+   To choose a specific Minikube driver, please use the `--bootstrap-flags vm-driver=xxx` command line parameter. For example to use the `kvm2` driver with clusterctl you would add `--bootstrap-flags vm-driver=kvm2`, for linux, if you haven't installed any driver, you can add `--bootstrap-flags vm-driver=none`.
+
+   - If you are using `Kind`:
+
+   ```bash
+   clusterctl create cluster \
+           --bootstrap-type kind --bootstrap-cluster-cleanup=false \
+           -c examples/_out/cluster.yaml \
+           -m examples/_out/machines.yaml \
```

Comment:
Since v1alpha2 patch got merged, I noticed that controlplane.yaml is generated instead of machines.yaml

Reply:
You're right and that's intended. I'll exchange machines.yaml with controlplane.yaml